Google safe browsing v2 (#12251)
demisto/content: Packs/GoogleSafeBrowsing/ReleaseNotes/1_1_0.md

#### Integrations
##### New: Google Safe Browsing v2

> - Added the **Google Safe Browsing v2 integration**. Previous functionality was maintained.

Reviewer: ```suggestion
- Added the **Google Safe Browsing v2 integration**. This integration is backward compatible.
```

Author: done
